vendor/github.com/rancher/rancher-compose-executor/rancher/wait.go

```diff
@@ -22,9 +22,21 @@ func (r *RancherService) WaitFor(resource *client.Resource, output interface{},
 }
 
 func (r *RancherService) Wait(service *client.Service) error {
-	return r.WaitFor(&service.Resource, service, func() string {
+
+	err := r.WaitFor(&service.Resource, service, func() string {
 		return service.Transitioning
```

Comment:
In `r.WaitFor` method it is already waiting for the service resource to become healthy(which waits its transition state). Why do we need to activate the service then apply the wait function again?